src/runtime/Types/ModuleObject.cs

```diff
@@ -132,6 +132,25 @@ public NewReference GetAttribute(string name, bool guess)
                 return new NewReference(c);
             }
 
+            // Simplified imported attribute names
+            // for attributes without the Attribute suffix, create an attribute builder.
+            // This means that imported attributes can be invoked using just e.g 'Browsable(False)'
+            // and not BrowsableAttribute(False) although its still an option.
+            var qname2 = qname + "Attribute";
+            var type2 = AssemblyManager.LookupTypes(qname2).FirstOrDefault(t => t.IsPublic);
+            if (type2 != null)
+            {
+                var str = "def {2}__AttributeBuilder(*args, **kwargs):\n"
+                          + "   import {1}\n"
+                          + "   return ({0}, args, kwargs)\n";
+                str = string.Format(str, qname2, _namespace, name);
+                PythonEngine.Exec(str);
+                using var attributeData = PythonEngine.Eval(name + "__AttributeBuilder");
+                var pinnedAttributeData = attributeData.NewReferenceOrNull();
+                StoreAttribute(name, pinnedAttributeData.Borrow());
+                return new NewReference(pinnedAttributeData);
+            }
+
```

Comment:
This seems overly complicated. Can't you just do what the code above does for type lookup, and just add another `StoreAttribute` for the alias?

Reply:
So for e.g BrowsableAttribute, this generates a function which takes the arguments and stores it as a tuple along with the type name. That way we can reconstruct which arguments are used for creating the attribute when later generating the code.

The reason it needs to be a function is that whenever e.g `Browsable`is invoked, it needs to generate a new tuple because the arguments are not the same every time.